{"identifier":{"url":"doc:\/\/com.TransmitSecurity.IdentityVerification\/documentation\/IdentityVerification\/TSIdentityVerificationError\/verificationStatusError","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/identityverification\/tsidentityverificationerror\/verificationstatuserror"]}],"abstract":[{"type":"text","text":"An unexpected technical error occurred (e.g., communication error)"}],"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"verificationStatusError","kind":"identifier"}]}],"kind":"declarations"}],"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"verificationStatusError","kind":"identifier"}],"externalID":"s:20IdentityVerification010TSIdentityB5ErrorO018verificationStatusD0yA2CmF","title":"TSIdentityVerificationError.verificationStatusError","roleHeading":"Case","modules":[{"name":"IdentityVerification"}],"role":"symbol","symbolKind":"case"},"schemaVersion":{"major":0,"patch":0,"minor":3},"kind":"symbol","sections":[],"hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.IdentityVerification\/documentation\/IdentityVerification","doc:\/\/com.TransmitSecurity.IdentityVerification\/documentation\/IdentityVerification\/TSIdentityVerificationError"]]},"references":{"doc://com.TransmitSecurity.IdentityVerification/documentation/IdentityVerification/TSIdentityVerificationError":{"role":"symbol","title":"TSIdentityVerificationError","abstract":[{"type":"text","text":"List of possible SDK errors"}],"kind":"symbol","type":"topic","url":"\/documentation\/identityverification\/tsidentityverificationerror","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"TSIdentityVerificationError","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"TSIdentityVerificationError"}],"identifier":"doc:\/\/com.TransmitSecurity.IdentityVerification\/documentation\/IdentityVerification\/TSIdentityVerificationError"},"doc://com.TransmitSecurity.IdentityVerification/documentation/IdentityVerification/TSIdentityVerificationError/verificationStatusError":{"url":"\/documentation\/identityverification\/tsidentityverificationerror\/verificationstatuserror","type":"topic","abstract":[{"text":"An unexpected technical error occurred (e.g., communication error)","type":"text"}],"title":"TSIdentityVerificationError.verificationStatusError","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"verificationStatusError"}],"kind":"symbol","identifier":"doc:\/\/com.TransmitSecurity.IdentityVerification\/documentation\/IdentityVerification\/TSIdentityVerificationError\/verificationStatusError"},"doc://com.TransmitSecurity.IdentityVerification/documentation/IdentityVerification":{"type":"topic","url":"\/documentation\/identityverification","title":"IdentityVerification","kind":"symbol","abstract":[],"identifier":"doc:\/\/com.TransmitSecurity.IdentityVerification\/documentation\/IdentityVerification","role":"collection"}}}